validator/webui/index.html

```diff
@@ -252,7 +171,9 @@
         selectAndFocusError: function(index) {
           this.$.listbox.selected = index;
           this.$.listboxItems.render();
-          this.$.listbox.selectedItem.scrollIntoView();
+          if (this.$.listbox.selectedItem) {
+            this.$.listbox.selectedItem.scrollIntoView();
+          }
```

Comment:
I imagine this results from setting selected to an invalid index. Should we do another check further up before that assignment as well?